plugin/net/driver.go

```diff
@@ -206,7 +211,7 @@ func (driver *driver) DiscoverDelete(disco *api.DiscoveryNotification) error {
 }
 
 func vethPair(id string) (string, string) {
-	return "vethwl" + id[:5], "vethwg" + id[:5]
+	return "vethwl" + id[:8], "vethwg" + id[:8]
```

Comment:
Where does the 14 character limit come from? A comment with a link explaining that would be really helpful here.